docs/markdown/howtox.md

```diff
@@ -211,18 +211,18 @@ test failures.
 
 ## Use Clang static analyzer
 
-Install scan-build program, then do this:
+Install scan-build program, then do this according to the [scan-build website](https://clang-analyzer.llvm.org/scan-build.html):
 
 ```console
 $ meson setup builddir
-$ ninja -C builddir scan-build
+$ scan-build ninja -C builddir
```

Comment:
This is not the same. Meson has a builtin target called `scan-build` which sets up a fresh build directory from scratch and runs scan-build on it.